nipype/algorithms/modelgen.py

```diff
@@ -85,7 +85,10 @@ def spm_hrf(RT, P=None, fMRI_T=16):
     if P is not None:
         p[0:len(P)] = P
 
+    old_settings = np.seterr(divide='ignore')  # Known division-by-zero
```

Comment:
could you rephrase this to:

	with np.errstate(divide='ignore'):
	
See: http://stackoverflow.com/questions/25087769/runtimewarning-divide-by-zero-error-how-to-avoid-python-numpy